src/Services/TaskLinkService.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+require_once 'src/Services/DatabaseService.php';
+$db = DatabaseService::connect();
+
+$tasksModel = new TasksModel($db);
```

Comment:
This file isn't a service - it's just some code that's in a different file. 

I like that you've tried to tidy up project.php by moving this stuff out of it - but we don't really have any proper place for this kind of code to go - so for now just move it back into the file that needs it.

It makes the code in project.php less clear because there are variables used in that file that are coming from this service.

Once we have Laravel we'll have a home for this kind of code